packages/kubernetes-1.24/kubernetes-1.24.spec

```diff
@@ -85,6 +85,7 @@ cp third_party/forked/golang/LICENSE LICENSE.golang
 cp third_party/forked/golang/PATENTS PATENTS.golang
 
 %build
+export FORCE_HOST_GO=1
```

Comment:
Agreed about putting them in their own commit (`kubelet: set FORCE_HOST_GO=1 in environment`).

But it does need to be set here, before `make generated_files` is invoked, or it will try again to download the Go toolchain for codegen.

Reply:
Oh interesting, I didn't realize they used go in the codegen :| Ok, then I agree that is where it should be.